.github/workflows/api-server-tests.yml

```
@@ -61,20 +62,19 @@ jobs:
           - k8s_version: "v1.29.2"
             cache_enabled: "true"
-            argo_version: "v3.5.15"
           - k8s_version: "v1.31.0"
             cache_enabled: "true"
-            pod_to_pod_tls_enabled: "true"
+            pod_to_pod_tls_enabled: "false"
+          - k8s_version: "v1.31.0"
+            proxy: "true"
+            pod_to_pod_tls_enabled: "false"
       fail-fast: false # So that failure in 1 type of parameterized job does not cause other jobs to terminate prematurely
     name: KFP API Server tests Standalone - K8sVersion=${{ matrix.k8s_version }} argo_version=${{ matrix.argo_version }} cacheEnabled=${{ matrix.cache_enabled }} proxyEnabled=${{ matrix.proxy }} pod_to_pod_tls_enabled=${{ matrix.pod_to_pod_tls_enabled }}
```

actionlint (1.7.10)

property "argo_version" is not defined in object type {cache_enabled: bool; k8s_version: string; pipeline_store: string; pod_to_pod_tls_enabled: bool; proxy: bool}

(expression)